akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/TestProbe.scala

```diff
@@ -154,8 +154,18 @@ object TestProbe {
   protected def expectMessageClass_internal[C](max: FiniteDuration, c: Class[C]): C
 
   /**
-   * Same as `receiveN(n, remaining)` but correctly taking into account
-   * the timeFactor.
+   * Receive one message of type `M` within the default timeout as deadline.
+   */
+  def receiveOne(): M
+
+  /**
+   * Receive one message of type `M`. Wait time is bounded by the `max` duration,
+   * with an [[AssertionError]] raised in case of timeout.
+   */
+  def receiveOne(max: Duration): M
```

Comment:
FiniteDuration instead of Duration

Reply:
I was going to do that but you specified Duration ;) changing.

Reply:
don't do what I say, but what I mean ;-)